src/cript/storage_clients/globus.py

```diff
@@ -6,8 +6,7 @@
 from globus_sdk.scopes import ScopeBuilder
 
 from cript.data_model.nodes.base_node import BaseNode
-from cript.storage_clients.exceptions import FileUploadError
-from cript.storage_clients.exceptions import FileDownloadError
+from .exceptions import InvalidAuthCode, FileUploadError, FileDownloadError
```

Comment:
'absolute imports' are preferred over 'relative imports'.
change to `from cript.exceptions import InvalidAuthCode, FileUploadError, FileDownloadError`